sdks/python/apache_beam/io/mongodbio.py

```diff
@@ -264,9 +264,11 @@ def display_data(self):
     res['uri'] = _mask_uri_password(self.uri)
     res['database'] = self.db
     res['collection'] = self.coll
-    res['filter'] = json.dumps(self.filter)
+    res['filter'] = json.dumps(
+        self.filter, default=lambda x: 'not_serializable')
```

Comment:
What's usually not serializable? non-JSON Python objects? Perhaps you can try `lambda x: 'not serializable(%s)' % str(x)`?

Reply:
(This is not required. Only up to you. Would it help if we add a unit test?)